src/components/Display.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+export default class Display extends React.Component {
+  render() {
+    return (
+      <div>{this.props.calcResult}</div>
+    );
+  }
+}
```

Comment:
- [ ] Make this a functional component
* [ ] it accepts `the result of the calculation` as an incoming prop ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/incomingProps.png))
* [ ] its result prop should be a String with default value = 0 ([should be String screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/validateAsString.png)) ([default props screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/display/DefaultProp.png))